content/en/apps/guides/hosting/android-app.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Android App Publishing"
+linkTitle: "Android App Publishing"
+weight: 
+description: >
+  How to building and publishing Android applications
+---
+
+## Building
+
+### Branding
+
+You can overwrite aspects of the generic Android app by providing your own brand. Common elements to overwrite are:
+
+- The URL of your CHT server so users don't have to type it in post install
+- The app logo and title
+- Translations for your supported languages
+
+Follow these [instructions to create a new brand](https://github.com/medic/medic-android/#adding-new-brands).
+
+### Building APKs
+
+Use Fastlane to build and sign your APKs ready for publishing. Building can be done using CI (eg: GitHub Actions, travis ci). For reference, refer to [how Medic builds APKs using GitHub Actions](https://github.com/medic/medic-android/blob/master/.github/workflows/publish.yml).
+
+It's important to ensure you sign your APKs using the same certificate to ensure that upgrading to a new version goes smoothly.
```

Comment:
maybe mention that the certificate is provided during the CI tool, and so this is why we don't recommend running fastlane locally to deply

Reply:
Added.